ICVL.py

```diff
@@ -0,0 +1,206 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from model_objects import model_parameters
+from DataSetInfoAbstractClass import dataSetInfoAbstract
+import seaborn as sns
+import cPickle
+
+
+class dataInfo(dataSetInfoAbstract):
+
+    def __init__(self):
+        self.name = "ICVL"
+
+    def load(self):
+
+        with open("Data/ICVL_Data/Training/ICVL_Training.pkl", "rb") as openfileobject:
```

Comment:
I'm not sure if I necessarily agree with making this a function parameter. This is a very specific implementation and specifically meant for the ICVL data. Because I'm going to write a script that will download, clean, and place it in a set location, the only reason this would have to be a parameter is if they move the folders around. If they want to use their own data, then they'll create their own implementation of the abstract class which will handle the loading of data there.

Another way to approach this would be to make the loading of data generally and not in the abstract function, but I'm not sure if I see an advantage in that because of how specifically most of the data needs to be treated. I originally processed all my data each time inside the load function (Before I pickled them), and if the user wants to add their own data, that will be very unique to their application.

Reply:
gotcha - ok, lets make some specific modifications then with that plan in mind.

How setting it to `self.training_file` in the `__init__` method? Also, should move towards the `os.path.join()` convention. This way a user can know where the file exists and it can be used later. Can then do `with open(self.training_file, 'rb') as openfileobject:...`

Reply:
can do for `self.testing_file` too